vendor/bat-native-ads/src/bat/ads/internal/ads/ad_events/new_tab_page_ads/new_tab_page_ad_event_handler.cc

```diff
@@ -136,8 +136,11 @@ void EventHandler::FireEvent(const NewTabPageAdInfo& ad,
             !ShouldRewardUser()) {
           // Fire an ad served event if Brave Ads are disabled and the ad
           // wasn't served by ads library.
-          FireEvent(placement_id, creative_instance_id,
-                    mojom::NewTabPageAdEventType::kServed);
+          const auto served_ad_event =
+              AdEventFactory::Build(mojom::NewTabPageAdEventType::kServed);
+          served_ad_event->FireEvent(ad);
```

Comment:
should we call ShouldDebounceAdEvent() for this event?

Reply:
No. We should only call for views and clicks